app/src/main/java/com/akp/ceg4110/quickreports/Incident.java

```diff
@@ -0,0 +1,72 @@
+package com.akp.ceg4110.quickreports;
+
+import android.graphics.Bitmap;
+
+import java.util.List;
+
+public class Incident {
+
+    private String name;
+    private String description;
+    private String weather;
+    private List<Bitmap> images;
+
+    //This will be the only constructor, the other ones are for debugging only
+    public Incident(String name, String description, String weather, List<Bitmap> images){
```

Comment:
If the constructor has all these, that means we are *forced* to have images. This might cause issues if we attempt to save without any of this information.

THe below constructors could be useful, but ideally I believe an empty constructor would work and then the subsequent items are added to the saved class upon clicking "Save".

Reply:
Ok. I think we can implement the savable method on everything but the name. Since the name is what I'm using as a primary key in the database and the requirements said that the incidents had to be searchable by name, it can't be null.

Reply:
Okay, that makes sense. On my end, I won't let the user Save without typing in a name.